### PR TITLE
Newsletter changes

### PR DESCRIPTION
**Pull request by rjackson**
After a chat with Joe about our newsletter and some desired changes to the membership system, I've put together this PR with the following changes:

- Remove newsletter checkbox from registration pages.
- Opt new registrations into newsletter emails by default
  - Under legitimate interest justification as we have an obligation to keep members of the space aware of news, announcements, and member meetings
  - If members are _really_ not interested in emails, we retain the newsletter checkbox on the edit account page to provide an opt-out mechanism
- Improve "Newsletter" control when editing account, to clarify its purpose and the options available
- Create a new "Newsletters" page
  - Accessible to admins only
  - Listing all e-mail addresses of newsletter recipients
  - Listing all e-mail addresses of active members

Whilst the membership system had a `/newsletters` endpoint with a JSON feed of newsletter opt-ins, the actual newsletter recipients were being retrieved by querying the database directly. We have repurposed that endpoint for the "newsletters" page, and updated its underlying conditions to include active or recently lapsed (within the last 6 months) members.

Screenshot of the new newsletters page:

Screenshot 2022-12-01 at 20 01 08

Screenshot of the newsletter control on the "Edit account" page:

image

cc @josephxtian – as I can't seem to add you as a reviewer 😢

### CODE REVIEW

**Review by conorriches — APPROVED** [review]
Great stuff!

File: resources/views/account/edit.blade.php [hunk 1]
```diff
@@ -165,14 +165,15 @@
...
 </div>
 
-<h2 id="newsletter">Newsletter Opt-In</h4>
+<h2 id="newsletter">Newsletter</h4>
 <div class="row">
     <div class="col-xs-12 col-md-8">
         <div class="form-group {{ Notification::hasErrorDetail('newsletter', 'has-error has-feedback') }}">
             {!! Form::checkbox('newsletter', true, null, ['class'=>'']) !!}
-            {!! Form::label('newsletter', 'Opt in to occasional newsletters', ['class'=>'']) !!}
+            {!! Form::label('newsletter', 'Opt in to membership newsletters', ['class'=>'']) !!}
```
Comment (chris18890) [inline resources/views/account/edit.blade.php:173]: Should this be saying "Opt in" if members are already opted in?

**Review by chris18890 — CHANGES_REQUESTED** [review]
Should this be saying "Opt in" if members are already opted in?

File: resources/views/account/edit.blade.php (hunk 1 above)
Comment (rjackson) [reply resources/views/account/edit.blade.php:173]: I _think_ it still worked in context, but yeah there are tweaks I want to explore to make this clearer to members.

I've been playing around. What do we think of the following?

image

Comment (chris18890) [reply resources/views/account/edit.blade.php:173]: Looks decent!

**Comment by josephxtian** [comment]
Could we change this to
Newsletters will be sent out periodically to keep you up to date with
announcements, news and events relevant to your membership of the space.

Just because the newsletter isn't quarterly, and by not defining the period
we never need to change the text again.

Cheers :)

**Review by chris18890 — APPROVED** [review]

**Pull request merged**